### PR TITLE
modulesync 1.9.6

### PR DESCRIPTION
**Pull request by bastelfreak**
modulesync 1.9.5

### CODE REVIEW

**Comment by bastelfreak** [comment]
debug output from the testrun:

```
  Debug: Augeas[rundeck/web.xml/security-role/role-name](provider=augeas): Opening augeas with root /, lens path , flags 64
  Debug: Augeas[rundeck/web.xml/security-role/role-name](provider=augeas): Augeas version 1.10.1 is installed
  Debug: Augeas[rundeck/web.xml/security-role/role-name](provider=augeas): Will attempt to save and only run if files changed
  Debug: Augeas[rundeck/web.xml/security-role/role-name](provider=augeas): sending command 'set' with params ["/files/var/lib/rundeck/exp/webapp/WEB-INF/web.xml/web-app/security-role/role-name/#text", "user"]
  Debug: Augeas[rundeck/web.xml/security-role/role-name](provider=augeas): Closed the augeas connection
  Error: /Stage[main]/Rundeck::Config::Global::Web/Augeas[rundeck/web.xml/security-role/role-name]: Could not evaluate: Saving failed, see debug
  Debug: Augeas[rundeck/web.xml/session-config/session-timeout](provider=augeas): Opening augeas with root /, lens path , flags 64
  Debug: Augeas[rundeck/web.xml/session-config/session-timeout](provider=augeas): Augeas version 1.10.1 is installed
  Debug: Augeas[rundeck/web.xml/session-config/session-timeout](provider=augeas): Will attempt to save and only run if files changed
  Debug: Augeas[rundeck/web.xml/session-config/session-timeout](provider=augeas): sending command 'set' with params ["/files/var/lib/rundeck/exp/webapp/WEB-INF/web.xml/web-app/session-config/session-timeout/#text", "30"]
  Debug: Augeas[rundeck/web.xml/session-config/session-timeout](provider=augeas): Closed the augeas connection
  Error: /Stage[main]/Rundeck::Config::Global::Web/Augeas[rundeck/web.xml/session-config/session-timeout]: Could not evaluate: Saving failed, see debug
  Debug: Augeas[rundeck/web.xml/security-constraint/auth-constraint/role-name](provider=augeas): Opening augeas with root /, lens path , flags 64
  Debug: Augeas[rundeck/web.xml/security-constraint/auth-constraint/role-name](provider=augeas): Augeas version 1.10.1 is installed
  Debug: Augeas[rundeck/web.xml/security-constraint/auth-constraint/role-name](provider=augeas): Will attempt to save and only run if files changed
  Debug: Augeas[rundeck/web.xml/security-constraint/auth-constraint/role-name](provider=augeas): sending command 'set' with params ["/files/var/lib/rundeck/exp/webapp/WEB-INF/web.xml/web-app/security-constraint[last()+1]/auth-constraint/role-name/#text", "*"]
  Debug: Augeas[rundeck/web.xml/security-constraint/auth-constraint/role-name](provider=augeas): Closed the augeas connection
  Error: /Stage[main]/Rundeck::Config::Global::Web/Augeas[rundeck/web.xml/security-constraint/auth-constraint/role-name]: Could not evaluate: Saving failed, see debug
```

**Comment by bastelfreak** [comment]
I am not really sure why augeas fails to work properly anymore :(

**Comment by fe80** [comment]
In the release 3.x the web.xml conf file is not available:
```
The web.xml file is no longer available. If you were modifying this after install before, let us know how/for what reason. (Typically modifying session timeout or auth constraints). Also, please see Authentication Changes section.
```

http://rundeck.org/docs/history/changelog.html

I've rollback to last 2.x version. I think this class need a review for 3.x ?

File: spec/acceptance/rundeck_spec.rb [hunk 1]
```diff
@@ -7,14 +7,16 @@
       class { 'java':
         distribution => 'jre'
       }
-      class { 'rundeck': }
+      class { 'rundeck':
+        package_ensure => '2.9.4'
```
Comment (philippeganz) [inline spec/acceptance/rundeck_spec.rb]: Why version 2.9.4 if I may ask ?

Comment (bastelfreak) [reply spec/acceptance/rundeck_spec.rb]: That was the latest version of the 2.x branch that I could find on their mirrors.

Comment (philippeganz) [reply spec/acceptance/rundeck_spec.rb]: I'd say 2.11.5 is the latest 2.x from [the release page](http://rundeck.org/news/2018/07/06/rundeck-2.11.5.html) and available on [their repo](https://rundeck.bintray.com/rundeck-rpm/:rundeck-2.11.5-1.56.GA.noarch.rpm). I guess we should try to run the tests against the latest 2.x, although 2.9.4 is probably fine :)

Comment (bastelfreak) [reply spec/acceptance/rundeck_spec.rb]: ups sorry. I will update it later.

**Pull request merged**